extension.ts

```diff
@@ -12,6 +12,7 @@ import { showCmdLine } from './src/cmd_line/main';
 import { ModeHandler } from './src/mode/modeHandler';
 import { TaskQueue } from './src/taskQueue';
 import { Position } from './src/motion/position';
+import './src/globals';
```

Comment:
Instead of importing everything from globals into the namespace, can you do something like

`import { globals } from './src/globals'` 

and then check `globals.isTesting`?